test/TensorNetwork_test.jl

```diff
@@ -319,6 +415,26 @@
         end
     end
 
+    @testset "Base.in" begin
+        @testset "by tensor" begin
+            tensor1 = Tensor(rand(3, 4), (:i, :j))
+            tensor2 = Tensor(rand(4, 5), (:j, :k))
+            tn = TensorNetwork([tensor1, tensor2])
+
+            # Broadcast not working
+            @test in(tensor1, tn)
+            @test in(tensor2, tn)
```

Comment:
You can use `Base.Fix2` if you want to fix the second argument for broadcasting in the first:
```suggestion
            @test all(broadcast(Base.Fix2(in, tn), [tensor1, tensor2]))
```

Reply:
I would write it like this:
```suggestion
            @test all(∈(tn), [tensor1, tensor2])
```